cfgov/jinja2/v1/_includes/molecules/social-media.html

```diff
@@ -21,20 +21,23 @@
                              to be appended to default Tweet text.
                              https://dev.twitter.com/web/tweet-button/parameters
 
-   options.twitter_related: A comma-separated list of accounts
-                            related to the content of the shared URI.
-                            https://dev.twitter.com/web/tweet-button/parameters
+   options.twitter_related:  A comma-separated list of accounts
+                             related to the content of the shared URI.
+                             https://dev.twitter.com/web/tweet-button/parameters
+                             Default is `cfpb`.
 
-   options.twitter_lang:    Loads text components in the specified language.
-                            https://dev.twitter.com/web/tweet-button/parameters
+   options.twitter_lang:     Loads text components in the specified language,
+                             if other than English.
+                             https://dev.twitter.com/web/tweet-button/parameters
 
    ========================================================================== #}
 
 {# TODO: Remove global page reference when moving to Wagtail. #}
-{% macro render(is_share_view=true, options={}) %}
+{% macro render( options={} ) %}
 {% set title = options.title | urlencode or ( page or {} ).seo_title | urlencode  %}
```

Comment:
@kurtw @kave @richaagarwal The default title in tweets that don't set the title is blank. I'm guessing `( page or {} ).seo_title` is outdated somehow?